src/wallet/wallet.cpp

```diff
@@ -3891,7 +3891,9 @@ bool CWallet::CreateCoinStake(ChainstateManager& chainman, const CWallet* pwalle
                 // Add remainder to first output
                 txNew.vout.push_back(CTxOut(outValue + (i == 0 ? remainder : 0), scriptPubKeyOut));
                 // make sure transaction below 1kb
-                if (::GetSerializeSize(txNew, SER_NETWORK, PROTOCOL_VERSION) > 1024) {
+                if (::GetSerializeSize(txNew, SER_NETWORK, PROTOCOL_VERSION) > 1000) {
+                    if (!i)
+                        return error("CreateCoinStake : failed to create coinstake");
```

Comment:
until we bring in options to allow exceeding 1000 byte limit, i don't want to create any coinstakes over the limit, i don't think it's right, as we don't know by how much that limit has been exceeded.

Reply:
It shouldn't get exceeded if all is well with the code as it is now. I was just thinking about having code to ensure the coinstake is still created even if the code is to change in the future.

However, it should be fine if you'd like to resolve this and merge.

I think it would be a good idea to do a release candidate so it can be tried on testnet (or mainnet for those willing to try it there) for a little while before final release.